man/gbz80.7

```diff
@@ -754,22 +754,30 @@ Cycles: 1
 Bytes: 1
 .Pp
 Flags: None affected.
-.Ss JR e8
-Relative Jump by adding
-.Ar e8
-to the address of the instruction following the
-.Sy JR .
-To clarify, an operand of 0 is equivalent to no jumping.
+.Ss JR n16
+Relative Jump to address
+.Ar n16 .
+The address is encoded as a signed 8-bit offset from the end of the
+.Sy JR
+instruction, so the address must be between
+.Sy -128
+and
+.Sy 127
+bytes away.
+For example, the instruction
+.Ql Label: JR Label
```

Comment:
```suggestion
.Dl Label: jr Label
```

Reply:
I based this on other examples which use `.Ql` and uppercase:
```
This is sometimes written as
.Ql LDIO [n16],A ,
or
.Ql LD [$FF00+n8],A .
```

Reply:
Since this is a full assembly line (due to the label) and not just an instruction, I think it warrants a more out-of-line display. (Moreso since it slightly steps outside of pure instruction syntax.)

Also, should the label be on the same line as the instruction? If so, I think it would be best to at least double the spacing between the two.

Reply:
I felt like giving it a whole multi-line block would look excessive for explaining one simple instruction. Double spaces sounds good.